tests/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests.csproj

```diff
@@ -6,10 +6,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="$(RepoRoot)src\Aspire.Hosting\ContainerImageTags.cs" />
+    <Compile Include="..\Aspire.Npgsql.Tests\PostgreSQLContainerFixture.cs" />
     <None Include="$(RepoRoot)src\Components\Aspire.Npgsql.EntityFrameworkCore.PostgreSQL\ConfigurationSchema.json" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
(nit) let's be consistent on the order of the json file and the Compile items. These are in a different order than Aspire.Npgsql.Tests.